utils/restart.go

```diff
@@ -0,0 +1,26 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"log"
+)
+
+func HandleMissingWatchers(crdName string) {
```

Comment:
Please rename to: `HandleMissingWatch(reconciler, missingCRDName)`

Reply:
Is `reconciler` really required here? we are not going to use anything from it right?